.gitignore

```diff
@@ -291,5 +291,5 @@ $RECYCLE.BIN/
 # End of https://www.gitignore.io/api/osx,linux,python,windows,pycharm,visualstudiocode
 
 # Misc
-
-test_report
\ No newline at end of file
+test_report
+/.idea/*

```

Comment:
For some Reason, Idea would add files in a different folder, the current rule from gitignore would not work.

Reply:
no problem :)